storagemodule.py

```diff
@@ -0,0 +1,57 @@
+#Question: Do you want me to make this whole thing a module or lots of submodules ? 
+
+#Import libaries
+import os
+import shutil
+import earthpy as et
+
+#Make a directory in any os system's home directory 
+et.io.HOME
```

Comment:
so to make this more clear I would write a function called 
define setHomeDirectory():
   if os.path.exists(et.io.HOME):
      et.io.HOME
   else:
      throw error
then call it here instead of needing the comment.